src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs

```diff
@@ -8,7 +8,7 @@
 namespace System.Diagnostics.Eventing
 {
     [StructLayout(LayoutKind.Explicit, Size = 16)]
-    public struct EventDescriptor
+    public readonly struct EventDescriptor
```

Comment:
I'm not sure about changing public structs to readonly -- existing code that relies on the previous form may be affected

Reply:
This struct is immutable by design.